schemas/cs3/tbl/t_item/item.ksy

```diff
@@ -0,0 +1,109 @@
+meta:
+  id: item
+  endian: le
+  encoding: UTF-8
+  imports:
+    - /components/cs3/character
+    - /components/cs3/item_type
+    - /components/cs3/effect
+    - /components/common/element
+seq:
+  - id: item_id
+    type: u2
+  - id: character_restriction
+    type: character
+  - id: flags
+    type: strz
+  - id: item_type
+    type: item_type
+  - id: item_subtype
+    type: u2 # TODO: make an enum for this?
```

Comment:
Yes since the main type already had one, and it can act as documentation too

Reply:
done.